core/src/main/java/GameObjects/UrlBuilderWrapper.java

```diff
@@ -0,0 +1,70 @@
+package GameObjects;
+
+/**
+ * Created by Chris on 11/12/2016.
+ */
+
+public interface UrlBuilderWrapper {
```

Comment:
This should just be called "UrlBuilder", since it's not actually wrapping anything, but rather providing an interface.